setup.cfg

```diff
@@ -10,12 +67,10 @@ norecursedirs =
 addopts =
     --strict
     --color=yes
-
-[bdist_wheel]
-universal=1
-
-[metadata]
-license_file = LICENSE
+markers =
```

Comment:
I believe we can remove this markers, they are not used anymore. Possibly in a separated pr

Reply:
will be better in a separated PR

Reply:
@edgarriba wait, we do use markers, not? How we skip (some) of jit tests then? I also use them when run tests locally